charts/osm/templates/mutatingwebhook.yaml

```diff
@@ -17,3 +17,14 @@ webhooks:
   namespaceSelector:
     matchLabels:
       openservicemesh.io/monitored-by: {{.Values.OpenServiceMesh.meshName}}
+    matchExpressions:
+      # This label is explicitly set to ignore a namespace
+      - key: "openservicemesh.io/ignore"
+        operator: DoesNotExist
+
+      # This label is set by Helm when it creates a namespace (https://github.com/helm/helm/blob/release-3.2/pkg/action/install.go#L292)
+      # It ensures that pods in the control plane namespace are never injected with a sidecar
+      - key: "name"
```

Comment:
@shashankbarsin From the link referenced, this `name` label is set when `--create-namespace` is used to create the namespace via Helm. This is not true if we are installing into an existing namespace. We could consider deploying a namespace resource e.g. `osm-system` with the `openservicemesh.io/ignore` label as part of the chart. WDYT?

Reply:
I thought about this, we could very well create the namespace using a chart instead of using the Helm go client to auto-create the namespace as is the case now.

@ritazh would you mind raising an issue to handle the namespace creation in the cli as you have suggested and we can discuss over it?

Reply:
Added https://github.com/openservicemesh/osm/issues/1817 to track